README.md

```diff
@@ -316,8 +316,9 @@ jobs:
 ```
 ## Sample workflows for new traceability fields support 
 
- - Environment variable `HELM_CHART_PATHS` is a list of helmchart files used in k8s-bake and k8s-deploy
- - Use script to build image and add `dockerfile-path` label to it. The value expected is the link to the dockerfile. 
+ - Environment variable `HELM_CHART_PATHS` is a list of helmchart files expected by k8s-deploy - it can be populated from k8s-bake

```

Comment:
Suggestion:
``...it will be populated automatically if you are using `k8s-bake` to generate the manifests. ...``

Reply:
Done